galata/test/documentation/general.test.ts

```diff
@@ -625,7 +628,8 @@ test.describe('General', () => {
       '.lm-Menu ul[role="menu"] >> text=New Console for Notebook'
     );
 
-    await page.click('.jp-CodeConsole-input >> .cm-content');
+    await page.click('.jp-CodeConsole-input .jp-InputArea-editor');
+    await page.$('.jp-CodeConsole-input .jp-InputArea-editor.jp-mod-focused');
```

Comment:
Running out of ideas on this one (and unable to repro locally). It may be not a testing problem but a genuine race condition. If we want a quick fix, the worst case scenario is adding something like `sleep(200)`.

Reply:
Resolved in https://github.com/jupyterlab/jupyterlab/pull/14152.